src/foundry/common/data/data.mjs/tileData.d.ts

```diff
@@ -0,0 +1,217 @@
+import { TileOcclusion, TileOcclusionConstructorData } from './tileOcclusion';
+import { VideoData, VideoDataConstructorData } from './videoData';
+import { ConfiguredFlags, FieldReturnType, PropertiesToSource } from '../../../../types/helperTypes';
+import DocumentData from '../../abstract/data.mjs';
+import * as documents from '../../documents.mjs';
+import * as fields from '../fields.mjs';
+
+interface TileDataSchema extends DocumentSchema {
+  _id: typeof fields.DOCUMENT_ID;
+  img: typeof fields.VIDEO_FIELD;
+  width: typeof fields.REQUIRED_NUMBER;
+  height: typeof fields.REQUIRED_NUMBER;
+  x: typeof fields.REQUIRED_NUMBER;
+  y: typeof fields.REQUIRED_NUMBER;
+  z: FieldReturnType<typeof fields.INTEGER_FIELD, { required: true; default: 100 }>;
+  rotation: FieldReturnType<typeof fields.ANGLE_FIELD, { default: 0 }>;
+  alpha: typeof fields.ALPHA_FIELD;
+  tint: typeof fields.COLOR_FIELD;
+  hidden: typeof fields.BOOLEAN_FIELD;
+  locked: typeof fields.BOOLEAN_FIELD;
+  overhead: FieldReturnType<typeof fields.BOOLEAN_FIELD, { default: false }>;
+  occlusion: DocumentField<TileOcclusion> & {
+    type: typeof TileOcclusion;
+    required: false;
+    default: {};
+  };
+  video: DocumentField<VideoData> & {
+    type: typeof VideoData;
+    required: false;
+    default: {};
+  };
+  flags: typeof fields.OBJECT_FIELD;
+}
+
+interface TileDataProperties {
+  /**
+   * The _id which uniquely identifies this Tile embedded document
+   */
+  _id: string | null;
+
+  /**
+   * An image or video file path which this tile displays
+   */
+  img: string | null | undefined;
+
+  /**
+   * The pixel width of the tile
+   * @defaultValue `0`
+   */
+  width: number;
+
+  /**
+   * The pixel height of the tile
+   * @defaultValue `0`
+   */
+  height: number;
+
+  /**
+   * The x-coordinate position of the top-left corner of the tile
+   * @defaultValue `0`
+   */
+  x: number;
+
+  /**
+   * The y-coordinate position of the top-left corner of the tile
+   * @defaultValue `0`
+   */
+  y: number;
+
+  /**
+   * The z-index ordering of this tile relative to its siblings
+   * @defaultValue `100`
+   */
+  z: number;
+
+  /**
+   * The angle of rotation for the tile between 0 and 360
+   * @defaultValue `0`
+   */
+  rotation: number;
+
+  /**
+   * @defaultValue `1`
+   */
+  alpha: number;
```

Comment:
`alpha`, `tint`, `overhead`, `occlusion`, and `video` seem to be missing from foundry's documentation, consider creating an issue.